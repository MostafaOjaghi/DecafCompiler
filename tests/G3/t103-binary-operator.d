void main () {
	int a;
	int b;
	a = ReadInteger();
	b = ReadInteger();
	Print(a + 10);
	Print(a + b);
	Print(a - b);
	Print(a * b);
	Print(a / b);
	Print(a % b);
}