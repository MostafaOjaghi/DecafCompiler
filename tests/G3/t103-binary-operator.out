38
33
23
140
5
3
